app/Domain/Goalcanvas/Controllers/ShowCanvas.php

```diff
@@ -265,10 +64,41 @@ public function run()
             $this->tpl->assign('allCanvas', $allCanvas);
             $this->tpl->assign('canvasItems', $this->goalService->getCanvasItemsById($currentCanvasId));
             $this->tpl->assign('users', $this->projectService->getUsersAssignedToProject(session("currentProject")));
+        
+            return $this->tpl->display(static::CANVAS_NAME . 'canvas.showCanvas');
```

Comment:
Since this is the goal canvas you can change "static::CANVAS_NAME" to 'goalcanvas.showCanvas'